app/controllers/mixins/ems_common_angular.rb

```diff
@@ -110,6 +112,16 @@ def create_ems_button_validate
       update_ems_button_validate(ems)
     end
 
+    def add_proxy_custome_attributes(ems, params)
+      %w(no_proxy http_proxy https_proxy).each do |att|
+        key = "container_" + att
+        ems.custom_attributes.create(
+          :name  => att.upcase,
+          :value => params[key]
```

Comment:
if we are dropping the UI part then we will drop this too..  Otherwise we probably can find some place inside the provider scope, but I couldn't and was hoping to get a hint.